cmd/snap/cmd_snap_op.go

```diff
@@ -680,6 +684,8 @@ type cmdRefresh struct {
 	IgnoreValidation bool                   `long:"ignore-validation"`
 	IgnoreRunning    bool                   `long:"ignore-running" hidden:"yes"`
 	Transaction      client.TransactionType `long:"transaction" default:"per-snap" choice:"all-snaps" choice:"per-snap"`
+	Hold             string                 `long:"hold" optional:"yes" optional-value:"forever"`
+	Resume           bool                   `long:"resume"`
```

Comment:
this needs to be unhold

Reply:
Ah sorry, I actually renamed all of the "resumes" to "unholds" after the meeting but forgot to force-push. I'll do that now but it might break the review state on Github